README.md

```diff
@@ -429,9 +429,9 @@ Hurray! You successfully made a contribution!
   - [Taneem Jan](https://github.com/taneemishere)
   - [Tanya Powell](https://github.com/tanyapowell)
   - [Tawfik Yasser](https://github.com/TawfikYasser)
+  - [Tirtharaj Sinha](https://github.com/tirtharajsinha)
```

Comment:
It looks good to me, but line 435 must be blank (which means that make a blank space between text and heading).